src/work-allocation-2/containers/info-message-container/info-message-container.component.ts

```diff
@@ -32,8 +33,14 @@ export class InfoMessageContainerComponent implements OnInit {
 
   public ngOnInit(): void {
     this.router.events.subscribe(event => {
+      if (event instanceof NavigationStart) {
+        // keep the current url the navigation started from
+        this.currentUrl = this.router.url;
+      }
       if (event instanceof NavigationEnd) {
-        this.resetMessages();
+        if (!this.currentUrl.includes('#manage')) {
```

Comment:
I udnerstand your logic. But, for completeness sake  @connorpgpmcelroy  could you please add a comment why we need to reset. Thanks

Reply:
will do